### PR TITLE
data: rahulj9a

### PR DESCRIPTION
**Pull request by Rahulj9a**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by Viveksati5143 — APPROVED** [review]

File: data/rahulj9a/testimonials/devfolio.json
Comment (SaraJaoude) [inline data/rahulj9a/testimonials/devfolio.json:4]: The word `Date` should have a lower case `d`.

Your file name needs to match your GitHub username exactly (it is case sensitive) - you have put lower case `r`.

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Your file name needs to match your GitHub username exactly (it is case sensitive) - you have put lower case `r`.

**Comment by SaraJaoude** [comment]
@Viveksati5143 please do not approve a PR without firstly doing a proper review of it - this causes confusion to the author and maintainers.

**Comment by SaraJaoude** [comment]
You seem to have added a testimonial about you, on behalf of someone else.  Testimonials are meant to be created by the user who is giving them (in your case, devfolio).  They should also be created via the issue template (rather than manually as you have done) as we have a GitHub Action that will automate the process, so I will close this issue.

**Comment by Viveksati5143** [comment]
> @Viveksati5143 please do not approve a PR without firstly doing a proper review of it - this causes confusion to the author and maintainers.

Sorry I thought it is for testing purposes won't approve in future

**Pull request closed without merging**
